meteoroid/core/models.py

```diff
@@ -59,3 +59,13 @@ def __str__(self):
 
     class Meta:
         unique_together = ('orion_subscription_id', 'fiware_service', 'fiware_service_path')
+
+
+class Schedule(FIWAREBase):
+    trigger_name = models.CharField(max_length=64)
+    rule_name = models.CharField(max_length=64)
+    name = models.CharField(max_length=64)
+    function = models.ForeignKey(Function, on_delete=models.CASCADE, related_name='functions')
+
+    class Meta:
+        unique_together = ('fiware_service', 'fiware_service_path')
```

Comment:
I will get the following Error if I register a different schedule twice. The fiware_service and fiware_service_path of the two schedulers are the same and have different names.
Please make sure unique constraint.
`AssertionError: You cannot call .save() on a serializer with invalid data.`

Reply:
Thank you for trying creating multiple schedules resource.
I added name key to unique_together.